### ISSUE TITLE
Next.js 14 only?

### ISSUE THREAD

**Issue by fmerian** [unicodeveloper/awesome-nextjs#275]
this repo is such a great source of knowledge! 

how about keeping it up to date with resources that only support the latest version of Next.js (currently: Next.js 14)?

**Comment by uniquejava** [comment]
Agree, these `awesome*` projects should have versioning(tagging). It's not beginner friendly and somewhat wasting time in such info-explosion era.
